crypto/encrypt.go

```diff
@@ -36,6 +36,7 @@ func GenerateTwofishKey() (key TwofishKey, err error) {
 // NewCipher creates a new Twofish cipher from the key.
 func (key TwofishKey) NewCipher() cipher.Block {
 	// NOTE: NewCipher only returns an error if len(key) != 16, 24, or 32.
+	//       Assumption tested in TestTwofishNewCipherAssumption.
```

Comment:
this comment doesn't add anything, and also if you are treating your dependencies as a black box you can't be certain that there aren't other reasons for an error to be returned (such as 'NoEntropy' or 'CryptoDoesn'tWorkOnTuesdays').